client/src/components/ui/RowTemplate.svelte

```diff
@@ -6,17 +6,22 @@
     import OverflowMenuVertical from '../icons/OverflowMenuVertical.svelte';
 
     export let iconSize: IconSize = IconSize.Large;
-    export let title: string;
+    export let showOverflowIcon = true;
 
     const dispatch = createEventDispatcher();
 </script>
 
-<article>
+<article on:keydown on:click|stopPropagation={() => dispatch(Events.RowContainerClick)}>
     <div class="icon"><slot name="icon" /></div>
-    <p>{title}</p>
-    <div class="icon" on:keydown on:click|stopPropagation={() => dispatch(Events.OverflowClick)}>
-        <OverflowMenuVertical size={iconSize} alt="Show overflow menu" />
+    <div id="middle">
+        <div><slot/></div>
+        <div class="subtext"><slot name="secondary"/></div>
```

Comment:
Self-closing `<slot>` elements are [not allowed](https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element). That is, the open and closing tags are mandatory.
```suggestion
        <div><slot></slot></div>
        <div class="subtext"><slot name="secondary"></slot></div>
```